### PR TITLE
Close tabs - use mouse middle button click and close by tab title

### LINKED ISSUE #12: Feature: When CloseTabs is enabled. Able to use mouse middle button click to close the tab.
Just like it happens with normal browser tabs.

### PR DESCRIPTION
**Pull request by Joni-Torrado**
Closes #12

### CODE REVIEW

File: src/XTabs/_XTabHeader.razor [hunk 1]
```diff
@@ -110,4 +110,10 @@
         _dropClass = string.Empty;
         return;
     }
+
+    private void CloseTabOnMouseWheel(XTab xTab, MouseEventArgs e)
+    {
+        if (e.Button != 1) return;
+        OnCloseTab.InvokeAsync(xTab);
```
Comment (David-Moreira) [inline src/XTabs/_XTabHeader.razor:117]: Async call. Can the method be turned into async?

**Review by David-Moreira — CHANGES_REQUESTED** [review]

**Comment by David-Moreira** [comment]
Closes #12

File: src/XTabs/_XTabHeader.razor (hunk 1 above)
Comment (Joni-Torrado) [reply src/XTabs/_XTabHeader.razor]: Yeah, no problem!

**Review by David-Moreira — APPROVED** [review]

**Review by David-Moreira — COMMENTED** [review]
We actually missed this: @onmousedown="(e) => CloseTabOnMouseWheel(XTab, e)"
Missing async call.

Fixed on latest dev150

**Pull request merged**